rpcd/playbooks/roles/elasticsearch/defaults/main.yml

```diff
@@ -31,8 +34,16 @@ elasticsearch_apt_repos:
 elasticsearch_apt_keys:
   - { url: "http://packages.elastic.co/GPG-KEY-elasticsearch", state: "present" }
 
+elasticsearch_version: 2.4.1
```

Comment:
Why are you requiring specific versions installed? In general we've not tended to do that. If it is necessary we need to look at tracking that so that it is updated on a regular schedule.